docs/UserGuide.md

```diff
@@ -492,29 +498,37 @@ Example:
 ![AvailableCommand](images/AvailableCommand.png)
 
 ### Command summary
+#### General command
```

Comment:
Should we add a line to introduce what the section is about?

Reply:
I don't think is necessary because I think the name is clear enough and the line might make it wordy. Let me know what do you think abt it.

Reply:
Ok, then I think its ok to leave it like that. Do you want to merge it then we can see how it looks 👍